apps/dashboard/src/components/stack-companion.tsx

```diff
@@ -2,386 +2,399 @@
 
 import { cn } from '@/lib/utils';
 import { checkVersion, VersionCheckResult } from '@/lib/version-check';
-import { Button } from '@stackframe/stack-ui';
+import { Button, Tooltip, TooltipContent, TooltipTrigger } from '@stackframe/stack-ui';
 import { BookOpen, HelpCircle, Lightbulb, TimerReset, X } from 'lucide-react';
-import { useEffect, useRef, useState } from 'react';
+import React, { createContext, useCallback, useContext, useEffect, useRef, useState } from 'react';
 import packageJson from '../../package.json';
 import { FeedbackForm } from './feedback-form';
 import { ChangelogWidget } from './stack-companion/changelog-widget';
 import { FeatureRequestBoard } from './stack-companion/feature-request-board';
 import { UnifiedDocsWidget } from './stack-companion/unified-docs-widget';
 
-type StackCompanionProps = {
-  className?: string,
-  onExpandedChange?: (expanded: boolean) => void,
-};
-
 type SidebarItem = {
   id: string,
   label: string,
   icon: React.ElementType,
   color: string,
+  hoverBg: string,
 };
 
-// Constants for resize constraints
-const MIN_SIDEBAR_WIDTH = 280;
-const MAX_SIDEBAR_WIDTH = 2000;
-const DEFAULT_SIDEBAR_WIDTH = 320;
-
-// Constants for cursor styles
-const CURSOR_STYLES = {
-  COL_RESIZE: 'col-resize',
-  DEFAULT: '',
-} as const;
-
-const USER_SELECT_VALUES = {
-  NONE: 'none',
-  DEFAULT: '',
-} as const;
-
 const sidebarItems: SidebarItem[] = [
   {
     id: 'docs',
     label: 'Docs',
     icon: BookOpen,
-    color: 'text-blue-600 hover:text-blue-700 dark:text-blue-400 dark:hover:text-blue-300',
+    color: 'text-blue-600 dark:text-blue-400',
+    hoverBg: 'hover:bg-blue-500/10',
   },
   {
     id: 'feedback',
     label: 'Feature Requests',
     icon: Lightbulb,
-    color: 'text-purple-600 hover:text-purple-700 dark:text-purple-400 dark:hover:text-purple-300',
+    color: 'text-purple-600 dark:text-purple-400',
+    hoverBg: 'hover:bg-purple-500/10',
   },
   {
     id: 'changelog',
     label: 'Changelog',
     icon: TimerReset,
```

Comment:
stack companion resizing feels flakey; only works sometimes, otherwise it'll close it